workspaces/src/network/result.rs

```diff
@@ -93,6 +157,12 @@ impl From<FinalExecutionOutcomeView> for CallExecutionDetails {
                     .iter()
                     .map(|t| t.outcome.gas_burnt)
                     .sum::<u64>(),
+            transaction: transaction_result.transaction_outcome.into(),
+            receipts: transaction_result
+                .receipts_outcome
+                .into_iter()
+                .map(Into::into)
```

Comment:
```suggestion
                .map(ExecutionOutcome::from)
```

Optional: Might be more readable if these conversion types are specified here